### PR TITLE
Django19 compatibility

### PR DESCRIPTION
**Pull request by martinphellwig**

### CODE REVIEW

File: glitter/blockadmin/blocks.py [hunk 1]
```diff
@@ -105,6 +105,9 @@ def inner(request, *args, **kwargs):
         inner = csrf_protect(inner)
         return update_wrapper(inner, view)
 
+    def get_app_list(self, request):
+        return dict()
```
Comment (tomkins) [inline glitter/blockadmin/blocks.py]: So that we're all on the same page, for consistency - let's use `{}` for all dicts.

File: glitter/utils.py [hunk 2]
```diff
@@ -101,3 +102,16 @@ def duplicate(obj, value=None, field=None, duplicate_order=None):
             if root_obj is None:
                 root_obj = obj
     return root_obj
+
+
+def django_version_gt_18():
```
Comment (tomkins) [inline glitter/utils.py]: Drop this function, just use:

```
if django.VERSION >= (1, 8)
```

In any code which needs to figure out the running version of Django.

Comment (martinphellwig) [inline glitter/utils.py]: Hmm didn't know that this would work.

**Comment by tomkins** [comment]
There's one thing I want to double check with rendering, but that's going to take a bit more time. I'd like to test it with a couple of sites locally as well to make sure everything still works.

Will analyse it more next week - but so far it looks good!

File: glitter/templatetags/glitter.py [hunk 3]
```diff
@@ -6,6 +6,19 @@
 register = template.Library()
 
 
+def _get_context_request(context):
```
Comment (tomkins) [inline glitter/templatetags/glitter.py]: This isn't the right solution to the problem - see below.

File: glitter/templatetags/glitter.py [hunk 4]
```diff
@@ -14,11 +27,14 @@ def glitter_head(context):
     with permission to edit the page.
     """
     user = context.get('user')
+    render_context, request = _get_context_request(context)
+    rendered = ''
 
     if user is not None and user.is_staff:
         template = get_template('glitter/include/head.html')
```
Comment (tomkins) [inline glitter/templatetags/glitter.py]: Try:

```
template = context.template.engine.get_template('glitter/include/head.html')
rendered = template.render(context)
```

Then you shouldn't need `_get_context_request`.

You might need another solution for multiple templates with the function below though - that I'll leave to you to figure out!

**Comment by tomkins** [comment]
Just one minor fix in 2 places, my comment from earlier was slightly off:

```python
if VERSION > (1, 8):
```

Should be:

```python
if VERSION >= (1, 9):
```

Otherwise Django 1.8.1 will be using the code you only want for 1.9.

Once you've fixed that, :shipit:

**Pull request merged**